src/lib/transition_router/transition_router.ml

```diff
@@ -138,34 +138,34 @@ let start_bootstrap_controller ~context:(module Context : CONTEXT) ~trust_system
           ~pipe_name:name ~logger ?valid_cb )
       ()
   in
-  transition_reader_ref := bootstrap_controller_reader ;
-  transition_writer_ref := bootstrap_controller_writer ;
```

Comment:
So, this line removal wasn't part of original fbc8bf0ff09

Reply:
I think I removed it because I was confused that the new code was redundant :
```
  let transition_writer_ref =
    Option.value transition_writer_ref
      ~default:(ref transition_frontier_controller_writer)
  in
  transition_writer_ref := transition_frontier_controller_writer ;
```
Was it a mistake ?

Reply:
I'll reply to https://github.com/MinaProtocol/mina/pull/16145/files#r1781433238, there you formulated your concern excellently!